<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="grb_Series">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>***</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QSplitter" name="splitter_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QPushButton" name="btn_AddSerie">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Add Serie</string>
                </property>
               </widget>
               <widget class="QComboBox" name="cmb_Sound">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
               <widget class="QGroupBox" name="grb_Radio">
                <property name="title">
                 <string>Series for Plot</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="MplWidget" name="MplWidget" native="true"/>
           <widget class="QTableWidget" name="tableWidget">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>13</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>15</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>16</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>wave samples</string>
             </property>
            </column>
           </widget>
          </widget>
          <widget class="QGroupBox" name="grb_Processing">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Processing</string>
           </property>
           <widget class="QLineEdit" name="led_begin">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="led_count">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>60</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>15</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_setarr">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>30</y>
              <width>136</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>Set Array Parameters</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cmb_Serie_in2">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>30</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
           </widget>
           <widget class="QComboBox" name="cmb_Serie_in1">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
           </widget>
           <widget class="QComboBox" name="cmb_Serie_out">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>30</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="led_single">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>60</y>
              <width>112</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_Open"/>
    <addaction name="act_Save"/>
    <addaction name="separator"/>
    <addaction name="act_Close"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="act_Plot"/>
    <addaction name="act_Table"/>
    <addaction name="act_Sound"/>
    <addaction name="act_Add_Serie"/>
    <addaction name="act_Addgrid"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="act_Filtering"/>
    <addaction name="act_Filtering_16"/>
    <addaction name="act_Minmax"/>
    <addaction name="act_Truba"/>
    <addaction name="act_Aproximate"/>
   </widget>
   <widget class="QMenu" name="menuPlotSeries">
    <property name="title">
     <string>Series for Plot</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSoundSeries">
    <property name="title">
     <string>Series for Sound</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="act_Add_arrays"/>
    <addaction name="act_Mult_const"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuOperations"/>
   <addaction name="menuPlotSeries"/>
   <addaction name="menuSoundSeries"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="act_Close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="act_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="act_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="act_Plot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="act_Sound">
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="act_Filtering">
   <property name="text">
    <string>Filtering</string>
   </property>
  </action>
  <action name="act_Filtering_16">
   <property name="text">
    <string>Filtering_16</string>
   </property>
  </action>
  <action name="act_Add_Serie">
   <property name="text">
    <string>Add Serie</string>
   </property>
  </action>
  <action name="act_Truba">
   <property name="text">
    <string>Truba</string>
   </property>
  </action>
  <action name="act_Minmax">
   <property name="text">
    <string>Minmax</string>
   </property>
  </action>
  <action name="act_Aproximate">
   <property name="text">
    <string>Aproximate</string>
   </property>
  </action>
  <action name="act_Table">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="act_Add_arrays">
   <property name="text">
    <string>Adding elements of two arrays</string>
   </property>
  </action>
  <action name="act_Mult_const">
   <property name="text">
    <string>Multiplying the elements of two arrays by the constant</string>
   </property>
  </action>
  <action name="act_Addgrid">
   <property name="text">
    <string>Add Grid to Plot</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
